add unsafeEscapeOptions documentation in neo4jGraphQL class (#301)
neo4j/docs-graphql: modules/ROOT/pages/neo4jgraphql-class.adoc

This is useful to prevent unbounded queries that can lead to performance issues.
See xref:security/avoid-unbounded-queries.adoc#_limit_required[Avoid unbounded queries > Require `limit` argument].

=== Unsafe escape options

[WARNING]
====
Changing these options is dangerous, and can lead to security risks.
====

> These are options intended to disable automatic escaping of input values in the generated Cypher. By default, node labels and relationship types are escaped in the generated Cypher, to ensure queries are safe.

Reviewer: ```suggestion
These are options intended to disable automatic escaping of input values in the generated Cypher.
By default, node labels and relationship types are escaped in the generated Cypher to ensure queries are safe.
```

> This behaviour can be disabled by using the flags inside `unsafeEscapeOptions`

Reviewer: ```suggestion
This behavior can be disabled by using the flags inside `unsafeEscapeOptions`:
```

> * `disableNodeLabelEscaping`: If set to true, node labels will not be escaped.

Reviewer: ```suggestion
* `disableNodeLabelEscaping`: If set to true, node labels are not escaped.
```

> * `disableRelationshipTypeEscaping`: If set to true, relationship types will not be escaped.

Reviewer: "false" is the default then?

Author: Yeah, I didn't put a default in part because we never technically set it as false, but yeah, we could say that is the default

Reviewer: if there is no default, how is the value set initially then?

Author: Well, it is javascript, so the value technically is `undefined`, we just treat that as false. It is just a technicality, do we want to put the default value in the docs?

Reviewer: the way it is currently written i think implies that it is without actually stating it. we might as well do so, if it is technically correct

> * `disableRelationshipTypeEscaping`: If set to true, relationship types will not be escaped.

Reviewer: ```suggestion
* `disableRelationshipTypeEscaping`: If set to true, relationship types are not escaped.
```
